### PR TITLE
Feature/52 Graph nodes styling

### PR DESCRIPTION
**Pull request by vidhya-metacell**

### CODE REVIEW

File: src/components/GraphViewer/GraphViewer.js [hunk 1]
```diff
@@ -1,52 +1,149 @@
...
+        hoverRectPosition[0] - 2,
+        hoverRectPosition[1] + hoverRectDimensions[1] + 2,
+      ];
+      const hoverRectBorderRadius = 2;
+      ctx.beginPath();
+      ctx.drawImage(
+        node.img,
+        node.x - size - 1,
+        node.y - size,
+        size * 2.4,
```
Comment (zsinnema) [inline src/components/GraphViewer/GraphViewer.js:104]: what is the 2.4 for?

Comment (zsinnema) [reply src/components/GraphViewer/GraphViewer.js:104]: perhaps it's a const value?

File: src/components/GraphViewer/GraphViewer.js [hunk 2]
```diff
@@ -1,52 +1,149 @@
...
-  }
+    graphRef.current.ggv.current.zoom(zoom - value, ONE_SECOND / 10);
+  };
 
   const resetCamera = (event) => {
     graphRef.current.ggv.current.zoomToFit();
-  }
+  };
 
   React.useEffect(() => {
```
Comment (jrmartin) [inline src/components/GraphViewer/GraphViewer.js:66]: Hi @vidhya-metacell, can you pass an empty array as a second parameter to the useEffect hook here so it behaves like componentDidMount? 

`React.useEffect(() => {
    setTimeout(
      () => graphRef?.current?.ggv?.current?.zoomToFit(),
      ONE_SECOND / 2
    );
  }, []);`

File: src/components/GraphViewer/GraphViewer.js (hunk 1 above)
Comment (vidhya-metacell) [reply src/components/GraphViewer/GraphViewer.js:104]: @zsinnema it's the image size of the node, yes a const

**Pull request merged**